cmd/get.go

```diff
@@ -296,6 +334,25 @@ var GetWfCmd = &cobra.Command{
 	},
 }
 
+func GetPrjIfPresent(cmd *cobra.Command) string {
```

Comment:
What do you think about `GetProjectID` as the name here? Abbreviating `project` to `prj` can be confusing.